app/views/base/captcha.scala.html

```diff
@@ -23,7 +23,7 @@
       @trans.clickOnTheBoardToMakeYourMove()
       <br /><br />
       @defining(netBaseUrl + routes.Round.watcher(captcha.gameId, captcha.white.fold("white", "black"))) { url =>
-      Help: <a class="hint--bottom" data-hint="View the solution" target="_blank" href="@url">@url</a>
+      @trans.help(): <a class="hint--bottom" data-hint="View the solution" target="_blank" href="@url">@url</a>
```

Comment:
Translation should include the colons ":" because it can work differently in some languages